### PR TITLE
Add plots

### PR DESCRIPTION
**Pull request by johpiip**
Added maps and graphs for some of the indicators. Maps use the basemap. Graphs include dummy data for scenario years because the purpose was to test different visualisations for bar plots. Everything is plotted in `/figures` folder.

In the future, data handling may need to be moved into separate file instead of doing it at first in every analysis file. Also, bar graphs may be drawn with Excel instead of R for easier modification for non-R folks.

### CODE REVIEW

File: scripts/figures/graph_mode-share_sustainable.R [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    panel.spacing = unit(0, "lines")
+  )
+
+ggsave(
+  here::here("figures", "graph_mode-share_sustainable.png"),
+  width = 148,
+  height = 105,
+  units = "mm",
+  dpi = 600
+)
```
Comment (zptro) [inline scripts/figures/graph_mode-share_sustainable.R:87]: Could this be parametrized? Seems to be the same settings everywhere.

File: scripts/figures/graph_mode-share_sustainable.R [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                      "helmet_4.0.4_2018_results",
+                      "origins_demand.txt") %>%
+  readr::read_tsv(
+    col_names = c("zone", "car", "transit", "bike", "walk"),
+    col_types = "idddd",
+    skip = 1
+  ) %>%
+  dplyr::select(zone, car, transit, bike, walk) %>%
+  tidyr::pivot_longer(cols = -zone, names_to = "mode", values_to = "demand") %>%
+  dplyr::mutate(mode = factor(mode, levels = c("car", "transit", "bike", "walk"), labels = c("Henkilöauto", "Joukkoliikenne", "Pyöräily", "Kävely")))
```
Comment (zptro) [inline scripts/figures/graph_mode-share_sustainable.R:20]: Make some kind of dictionary lookup for translations to Finnish?

**Review by zptro — APPROVED** [review]
Seems ok!

File: scripts/figures/graph_mode-share_sustainable.R (hunk 1 above)
Comment (johpiip) [reply scripts/figures/graph_mode-share_sustainable.R]: Great idea! I added this functionality, see here: https://github.com/HSLdevcom/mal-effect-calculations/pull/16/commits/d5f4ddebe7b5f7536f38f3a191ea74c7adf989ce

File: scripts/figures/graph_mode-share_sustainable.R (hunk 2 above)
Comment (johpiip) [reply scripts/figures/graph_mode-share_sustainable.R]: Awesome, I added this: https://github.com/HSLdevcom/mal-effect-calculations/pull/16/commits/f80fc1d4d05dfe75d5b8437f6bd3509d384185c0 I am still considering whether `utilities` folder is a good place or should I add everything to `data`... But I will change that later if needed.

**Comment by johpiip** [comment]
Thanks, @zptro! I will merge this now.

**Pull request merged**